dashboards.go

```diff
@@ -37,6 +37,39 @@ type GraphDefinitionRequest struct {
 	OrderBy        *string                            `json:"order_by,omitempty"`
 	ExtraCol       *string                            `json:"extra_col,omitempty"`
 	Metadata       map[string]GraphDefinitionMetadata `json:"metadata,omitempty"`
+
+	// For APM and log query
+	LogQuery *GraphApmOrLogQuery `json:"log_query,omitempty"`
```

Comment:
We actually need another one for APM query here using `apm_query` specifically to map properly to the API.
```
ApmQuery     *GraphApmOrLogQuery `json:"apm_query,omitempty"
````